.github/workflows/multi-arch-test-build.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Build
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: ${{ matrix.arch }} build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch:
+          - x86_64
+          - mips_24kc
+          - arm_cortex-a9_neon
+          - arm_cortex-a9_vfpv3-d16
+          - arc_arc700
+          - aarch64_generic
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 20
+      - run: git fetch --depth=20
```

Comment:
I remember doing a lot of tests for CircleCI and no numbers are enough to handle old PR, you need full checkout to be safe

Reply:
Done